spec/reader_spec.rb

```diff
@@ -1,34 +1,20 @@
-# coding: utf-8
+# frozen_string_literal: true
+
 require_relative 'spec_helper'
 require 'rdf/spec/reader'
 
 describe JSON::LD::Reader do
-  let!(:doap) {File.expand_path("../../etc/doap.jsonld", __FILE__)}
-  let!(:doap_nt) {File.expand_path("../../etc/doap.nt", __FILE__)}
-  let!(:doap_count) {File.open(doap_nt).each_line.to_a.length}
-  let(:logger) {RDF::Spec.logger}
+  let!(:doap) { File.expand_path('../etc/doap.jsonld', __dir__) }
+  let!(:doap_nt) { File.expand_path('../etc/doap.nt', __dir__) }
+  let!(:doap_count) { File.open(doap_nt).each_line.to_a.length }
+  let(:logger) { RDF::Spec.logger }
 
-  after(:each) {|example| puts logger.to_s if example.exception}
+  after { |example| puts logger if example.exception }
 
   it_behaves_like 'an RDF::Reader' do
-    let(:reader_input) {File.read(doap)}
-    let(:reader) {JSON::LD::Reader.new(reader_input)}
-    let(:reader_count) {doap_count}
-  end
-
-  describe ".for" do
```

Comment:
Why remove these tests? are they redundant?

Reply:
It's a mistake, thanks for pointing it out. I have rebased the branch.